GOVERNANCE.md

```diff
@@ -68,9 +68,23 @@ Members participate in a vote.
 
 ## Merging PRs into this Repository
 
-This section does not apply to [the Node.js core repository](https://github.com/nodejs/node).
-It only applies to [the bootstrap repository](https://github.com/nodejs/bootstrap).
+Pull requests that do not change the charter or governance of the CPC can be merged into
+this repository provided the following conditions have been met:
 
-Pull requests must have no objections and have been open for at least 72 hours 
-in order to be merged into this repository. A pull request that is unable to
-reach consensus cannot be merged into this repository.
+* there are no outstanding objections
+* there are two approvals by CPC members
+* the PR has been open for at least 72 hours
+
+Pull requests that change governance of the CPC must meet the following conditions
+in addition to the ones listed for regular PRs.
+
+* The PR has been open for at least 14 days OR has been approved by 50% or more
```

Comment:
I'd prefer "consensus in a meeting with quorum of voting members" rather than "50% approval"

Reply:
That sounds like it might be more appropriate as a third "OR" choice, rather than a replacement of the "50% approval" condition.

Reply:
I think we'd agree that this would be ok:

The PR has been open for at least 14 days OR consensus is reached in a meeting with quorum of voting members

Reply:
@MylesBorins updated with your suggestion.